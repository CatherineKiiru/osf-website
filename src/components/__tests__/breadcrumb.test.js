import React from "react";
import renderer from "react-test-renderer";

import Breadcrumb from "../2022/breadcrumb";

let wrapper;

describe("breadcrumb component", () => {
  beforeAll(() => {
    wrapper = renderer.create(<Breadcrumb name="test-name" />);
  });

  it("should have image tag with src attribute", () => {
    expect(wrapper.toJSON()).toMatchSnapshot();
  });

  it("should have text 'test-name'", () => {
    expect(wrapper.root.findByType("h1").children).toContain("test-name");
  });
});
