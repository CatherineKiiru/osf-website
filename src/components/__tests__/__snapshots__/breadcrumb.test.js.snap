// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`breadcrumb component should have image tag with src attribute 1`] = `
<div>
  <section
    className="flex flex-wrap md:flex-row bg-army-green-primary px-4 sm:px-8 lg:px-16 xl:px-40 2xl:px-64 pt-32 pb-20 sm:pb-6 md:pt-40 md:pb-32 lg:pt-60 xl:pb-24 2xl:pt-56 2xl:pb-48 sm:text-left"
  >
    <div
      className="w-full sm:w-1/2 md:w-1/2 lg:w-1/2 xl:w-1/2 mb-4 relative text-white"
    >
      <h1
        className="font-anisette text-4xl lg:text-6xl leading-tight font-bold"
      >
        test-name
      </h1>
    </div>
  </section>
</div>
`;
